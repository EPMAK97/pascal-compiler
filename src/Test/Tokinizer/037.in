2{aaa}3{aaa}