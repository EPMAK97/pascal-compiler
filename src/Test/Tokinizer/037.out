1  |  1 | INTEGER     | CONST_INTEGER               | 2         | 2
1  |  7 | INTEGER     | CONST_INTEGER               | 3         | 3
END OF FILE
